### PR TITLE
[Demo 2] header file fixed

### PR DESCRIPTION
**Pull request by Derrickakq**
commented out header files that do not exist in ros-melodic

### CODE REVIEW

File: exercises/Descartes_Planning_and_Execution/solution_ws/src/ur5_demo_descartes/include/ur5_demo_descartes/ur_moveit_plugin.h [hunk 1]
```diff
@@ -94,8 +94,8 @@
 #include <kdl/chainiksolvervel_pinv.hpp>
 #include <kdl/chainiksolverpos_nr_jl.hpp>
 #include <kdl/chainfksolverpos_recursive.hpp>
-#include <moveit/kdl_kinematics_plugin/chainiksolver_pos_nr_jl_mimic.hpp>
-#include <moveit/kdl_kinematics_plugin/chainiksolver_vel_pinv_mimic.hpp>
+//#include <moveit/kdl_kinematics_plugin/chainiksolver_pos_nr_jl_mimic.hpp>
+//#include <moveit/kdl_kinematics_plugin/chainiksolver_vel_pinv_mimic.hpp>
```
Comment (jrgnicho) [inline exercises/Descartes_Planning_and_Execution/solution_ws/src/ur5_demo_descartes/include/ur5_demo_descartes/ur_moveit_plugin.h]: can you delete these lines instead?

**Review by jrgnicho — CHANGES_REQUESTED** [review]

**Review by JeremyZoss — APPROVED** [review]
Looks good, easy change.  Builds on my PC.

For reference: this fix is confirmed in a MoveIt issue ([ros_planning/1351](https://github.com/ros-planning/moveit/issues/1351))

File: exercises/Descartes_Planning_and_Execution/solution_ws/src/ur5_demo_descartes/include/ur5_demo_descartes/ur_moveit_plugin.h (hunk 1 above)
Comment (JeremyZoss) [reply exercises/Descartes_Planning_and_Execution/solution_ws/src/ur5_demo_descartes/include/ur5_demo_descartes/ur_moveit_plugin.h]: fixed in commit 1a75c55.  Please re-review.

**Review by jrgnicho — APPROVED** [review]

**Comment by jrgnicho** [comment]
@JeremyZoss and @Derrickakq thanks for the contribution

**Pull request merged**